index.js

```diff
@@ -5,17 +5,15 @@ var _ = require("lodash");
 
 var envName = process.env.NODE_ENV || "development";
 var basePath = process.env.CONFIG_BASE_PATH || process.cwd();
+var configDir = process.env.CONFIG_DIR || "config";
```

Comment:
Base path is a documented feature:

https://github.com/ExpressenAB/exp-config#specifying-the-root-folder

Maybe the tests can use that feature instead. Having a temp folder as root with a "config" inside it?